src/plugins/image-plugin.tsx

```diff
@@ -63,18 +66,39 @@ export function ImagePlugin(): HtmlSerializablePlugin {
       configureImage: function (editor: Editor, displayDialog: DisplayDialogFunction) {
         displayDialog({
           title: "Insert Image",
-          prompts: ["Enter the URL of the image:"],
-          onAccept: (_editor, inputs) => _editor.command("addImage", inputs)
+          rows: [
+            { name: "source", type: "input", label: "Source URL:" },
+            { name: "description", type: "input", label: "Description:" },
+            { name: "dimensions", type: "label", label: "Dimensions:" },
+            [
+              { name: "width", type: "input", charSize: 6 },
+              { name: "x", type: "label", label: "x" },
+              { name: "height", type: "input", charSize: 6 },
+              { name: "spacer", type: "label", label: "\u00a0" },
+              { name: "constrain", type: "checkbox", label: "Constrain proportions" },
+            ],
+            {
+              name: "placement", type: "select", label: "Placement:",
+              options: [
+                { value: "inline", label: "Inline" },
+                { value: "float-left", label: "Float left" },
+                { value: "float-right", label: "Float right" },
+              ]
+            }
+          ],
```

Comment:
Is there any value in being able to turn off the advanced image settings (description, dimensions, etc.) and just present a dialog that requests the image URL?  Is it clear to a user that they don't need to insert image dimensions, for example?

Reply:
This mirrors the behavior of the existing editor. I don't think we have a use case for the simpler version at the moment, although that's not to say that the library shouldn't support that use case at some point. For the specific question you asked, the dimensions should auto-fill for the usual case of entering an image URL to an image that is available while the dialog is up.